openslides/agenda/main_menu.py

```diff
@@ -10,5 +10,5 @@ class AgendaMainMenuEntry(MainMenuEntry):
     verbose_name = ugettext_lazy('Agenda')
     required_permission = 'agenda.can_see'
     default_weight = 20
-    pattern_name = 'item_overview'
+    pattern_name = '/agenda'
```

Comment:
Ah, that's not a pattern_name but an absolute URL. Is there a possibility of naming URL patterns in AngularJS? If not we should rename this attribute.